docs/migration-v8.md

```diff
@@ -157,9 +157,9 @@ IAsyncPolicy retryPolicy = Policy.Handle<Exception>().WaitAndRetryAsync(3, _ =>
 IAsyncPolicy timeoutPolicy = Policy.TimeoutAsync(TimeSpan.FromSeconds(3));
 
 // Wrap the policies. The policies are executed in the following order (i.e. Last-In-First-Out):
-// 1. Retry
-// 2. Timeout
-IAsyncPolicy wrappedPolicy = Policy.WrapAsync(timeoutPolicy, retryPolicy);
+// 1. Retry << outer
+// 2. Timeout << inner
```

Comment:
It might be misunderstood as "much less than", so maybe this is better because (at least to me) it looks like an arrow.

```suggestion
// 1. Retry <== outer
// 2. Timeout <== inner
```